tests/random/prng_test.go

```diff
@@ -25,8 +27,65 @@ func deploy(t *testing.T) (*ethtest.SimulatedBackend, *TestablePRNG) {
 	return sim, prng
 }
 
-func TestRead(t *testing.T) {
-	_, prng := deploy(t)
+func deployCSPRNG(t *testing.T) (*ethtest.SimulatedBackend, *TestableCSPRNG) {
+	t.Helper()
+
+	sim := ethtest.NewSimulatedBackendTB(t, 1)
+	_, _, prng, err := DeployTestableCSPRNG(sim.Acc(0), sim)
+	if err != nil {
+		t.Fatalf("DeployTestablePRNG() error %v", err)
+	}
+
+	return sim, prng
+}
+
+type TestingContract interface {
+	Sample(opts *bind.CallOpts, seed [32]byte, bits uint16, n uint16) ([]*big.Int, error)
+	BitLength(opts *bind.CallOpts, n *big.Int) (*big.Int, error)
+	ReadLessThan(opts *bind.CallOpts, seed [32]byte, max *big.Int, n uint16) ([]*big.Int, error)
+	TestStoreAndLoad(opts *bind.TransactOpts, seed [32]byte, bits uint16, beforeStore uint16) (*types.Transaction, error)
+}
+
+type Implementation struct {
+	name string
+	sim  *ethtest.SimulatedBackend
+	prng TestingContract
+}
+
+func getImplementations(t *testing.T) []Implementation {
+	var impl []Implementation
+	{
```

Comment:
Here I may be a bit biased by c++, where I've seen this quite frequently. The reason why I used it here is because I was too lazy to keep track if I already declared a variable or not for `sim, prng := deploy...`. This allowed me to move around / comment out the blocks while developing without having to worry about using `:=` or `=`. Maybe there is another way for this?